src/database/mutations/Collection.ts

```diff
@@ -82,7 +83,11 @@ export async function createCollection(
   }
 
   // if a collection has any labels, set up a connection to labels, too
+  // first check if collection - label limit is not exceeded
```

Comment:
Nitpick: compound nouns as adjectives should be hyphenated, so "collection-label limit>